lib/spack/spack/test/packaging.py

```diff
@@ -39,9 +40,8 @@
 )
 from spack.spec import Spec
 
-pytestmark = pytest.mark.not_on_windows("does not run on windows")
-
 
+@pytest.mark.not_on_windows("gpg not available on Windows")
```

Comment:
The `mock_gnupghome` test fixture should automatically skip this test if gpg is not available, were you seeing an issue without this? Once gpg is added with https://github.com/spack/spack/pull/41810, I think it would be useful to start running this test.

Reply:
Yes. Without the pytest, the test results in an error message stating it can't bootstrap gpg.